web/wp-content/themes/dctx-v2/template-parts/content-page.php

```diff
@@ -0,0 +1,45 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-trjx8p.php, but found phpcs-scan-tRJX8P.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).